lib/streamlit/runtime/state/widgets.py

```diff
@@ -72,6 +72,15 @@
 )
 
 
+class DefaultValue:
```

Comment:
If we make this a dataclass, we can simplify some `isinstance` checks to `==` equality checks, which seems more semantically fitting.

Reply:
Oh, good point. I also just checked the spec and it specified to use ellipsis here instead of a class. I will check both options, change to dataclass or change to ellipsis tomorrow.